### PR TITLE
Always do force cleanup

### PR DESCRIPTION
**Pull request by zshi-redhat**
Closes #[USHIFT-324](https://issues.redhat.com//browse/USHIFT-324)

### CODE REVIEW

File: hack/cleanup.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -eu
+set -u
```
Comment (ggiguash) [inline hack/cleanup.sh:2]: I would not remove "fail on error" option. If you'd like a given command to always succeed, please add "|| true" to it.

For example:
crio wipe -f || true

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: hack/cleanup.sh (hunk 1 above)
Comment (zshi-redhat) [reply hack/cleanup.sh]: done

**Comment by oglok** [comment]
I don't understand, if we only use `crio wipe` we don't have control over the order of which pods will be removed first. If it happens to be the OVN ones, we'll hit the issue we had previously about `no ovn network found`. 

Am I missing something?

**Comment by zshi-redhat** [comment]
> I don't understand, if we only use `crio wipe` we don't have control over the order of which pods will be removed first. If it happens to be the OVN ones, we'll hit the issue we had previously about `no ovn network found`.
> 
> Am I missing something?

`crio wipe -f` force wipes all container images regardless of ovn pods status, together with `systemctl restart crio`, all running containers will be stopped and removed.  so this commit takes a force approach here to avoid cases that ovnk pods may not always be available.
 
For example: cleanup.sh script is executed after a reboot, not executed immediately followed by stopping microshift service, in such case, ovnk pods are not available to respond CNI request (from `crictl rmp/stopp`) which results in infinite loop.

**Comment by oglok** [comment]
> > I don't understand, if we only use `crio wipe` we don't have control over the order of which pods will be removed first. If it happens to be the OVN ones, we'll hit the issue we had previously about `no ovn network found`.
> > Am I missing something?
> 
> `crio wipe -f` force wipes all container images regardless of ovn pods status, together with `systemctl restart crio`, all running containers will be stopped and removed. so this commit takes a force approach here to avoid cases that ovnk pods may not always be available.
> 
> For example: cleanup.sh script is executed after a reboot, not executed immediately followed by stopping microshift service, in such case, ovnk pods are not available to respond CNI request (from `crictl rmp/stopp`) which results in infinite loop.

I see, to be honest I'm used to do cleanups right after I stop MicroShift's service and not after reboots. If this has been tested, I'm ok with merging it.

**Comment by dhellmann** [comment]
> I see, to be honest I'm used to do cleanups right after I stop MicroShift's service and not after reboots. If this has been tested, I'm ok with merging it.

I'm used to doing that, too. However, if the system isn't in exactly the right state when the current implementation is used then it doesn't work. If you run `cleanup.sh` twice in a row, for example, the second attempt hangs trying to kill the extra containers.

**Pull request merged**